SampleApp/SampleApp/View/Payments/PaymentsView.swift

```diff
@@ -0,0 +1,119 @@
+import SwiftUI
+import TyroTapToPaySDK
+
+enum InputFocus: Hashable {
+  case amountTextField
+  case paymentButton
+  case refundButton
+  case none
+}
+
+struct PaymentsView: View {
+  @State var amount: Decimal
+  @FocusState var amountInFocus: Bool
+  @State var activeFormatter: NumberFormatter
```

Comment:
why is this `@State`?

Reply:
> why is this `@State`?

Because the formatter used depends on the focus state,